attachment_image_resize/models/res_company.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2024 Quartile
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    attachment_image_resize_models = fields.Char(
+        help="When users attach images to these models, the images will be resized "
+        "based on the maximum resolution specified for attachments."
+    )
```

Comment:
If we keep this field, help text should contain an example of how it should be populated.

Reply:
I added the placeholder for this.